lms/migrations/versions/0a3bfcc14133_backfill_application_instance_id.py

```diff
@@ -0,0 +1,85 @@
+"""
+Bakfill application_instance_id in group_info, lis_result_sourcedid and oauth2_token.
+
+Revision ID: 0a3bfcc14133
+Revises: 407ff423b9e3
+Create Date: 2022-03-04 11:39:06.573269
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0a3bfcc14133"
+down_revision = "407ff423b9e3"
+
+
+def upgrade():
+    conn = op.get_bind()
+    # Fist backfill the new application_instance_id columns based on the existing consumer_key columns
+    conn.execute(
+        """
+            UPDATE group_info
```

Comment:
The three queries are pretty similar:
- Set application_instance_id from the  application_instances.id
- When the consumer_key on the table matches the one in application_instance
- Only for rows that don't have it already.